parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -1229,6 +1230,16 @@ impl_runtime_apis! {
 			Ok(batches)
 		}
 	}
+
+	impl sp_genesis_builder::GenesisBuilder<Block> for Runtime {
```

Comment:
Do we need `disable-genesis-builder` feature here? 
See [here](https://github.com/paritytech/polkadot/blob/6dfe98861dfe76c8c45297e310226865f02b0f9b/runtime/polkadot/src/lib.rs#L2196)